Add math rendering with Katex

rlee287 opened the pull request.

gicrisf commented:
Hi! Thank you for your PR! I'm sorry for the late response, but I'm drowning with the stuff to do in college, so I didn't have time. At a first glance, it looks like you completed the integration with LaTeX exactly like I planned to: with opt-in/out in config file and a per file basis. Am I right? The file you removed in the first commit was a macro of yours?

rlee287 commented:
The file that I removed was named "2", which exists in the `main` branch at `templates/macros/2` and which looked like a file accidentally left by someone else by a mistyped shell redirection. The theme worked for me after I removed it, but it should probably get a more descriptive name if it is supposed to do something.

gicrisf commented:
> The file that I removed was named "2", which exists in the `main` branch at `templates/macros/2` and which looked like a file accidentally left by someone else by a mistyped shell redirection. The theme worked for me after I removed it, but it should probably get a more descriptive name if it is supposed to do something.

Yes! You're absolutely right. I checked everything just now, sorry for the long wait. 
On the other hand, your macro is just fine, so I'm going to merge. 
Thanks again
